model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -56,6 +56,30 @@ data class AnalyzerResult(
         errors.any { it.value.isNotEmpty() }
                 || projects.any { it.scopes.any { it.dependencies.any { it.hasErrors() } } }
     }
+
+    /**
+     * A sorted set containing the [Identifier]s of all packages which are not excluded by the repository configuration.
+     */
+    private val includedPackageIds: SortedSet<Identifier> by lazy {
```

Comment:
Nit: Could drop the explicit return type.

Reply:
Done.